### PR TITLE
Add env file that pull arm64 images

### PR DESCRIPTION
**Pull request by JPWKU**
Signed-off-by: Jeremy Phelps <jphelps@linuxfoundation.org>

### CODE REVIEW

**Comment by JPWKU** [comment]
Just a start, we will have to build arm64 images for the other images listed in the env.

File: bin/arm64_env.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export volume=edgexfoundry/docker-edgex-volume
+export consul=edgexfoundry/docker-core-consul
+export coreMetadata=nexus3.edgexfoundry.org:10004/docker-core-metadata-go-arm64
+export mongo=edgexfoundry/docker-edgex-mongo
+export mongoSeed=edgexfoundry/docker-edgex-mongo-seed
```
Comment (feclare) [inline bin/arm64_env.sh]: mongo-seed is no longer needed. It was merged with docker-edgex-mongo.

File: bin/arm64_env.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+export supportLogging=nexus3.edgexfoundry.org:10004/docker-support-logging
```
Comment (feclare) [inline bin/arm64_env.sh]: Already available as go-arm64

File: bin/arm64_env.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+export coreData=nexus3.edgexfoundry.org:10004/docker-core-data-go-arm64
+export coreCommand=nexus3.edgexfoundry.org:10004/docker-core-command-go-arm64
+export supportScheduler=edgexfoundry/docker-support-scheduler
+export exportClient=edgexfoundry/docker-export-client
```
Comment (feclare) [inline bin/arm64_env.sh]: Already available as go-arm64

File: bin/arm64_env.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+export exportDistro=edgexfoundry/docker-export-distro
```
Comment (feclare) [inline bin/arm64_env.sh]: Already available as go-arm64

**Review by feclare — CHANGES_REQUESTED** [review]

**Review by feclare — APPROVED** [review]

**Review by tsconn23 — APPROVED** [review]

**Pull request merged**